ggml/src/ggml-vulkan/ggml-vulkan.cpp

```diff
@@ -1764,11 +1764,11 @@ static void ggml_vk_print_gpu_info(size_t idx) {
     fp16 = fp16 && vk12_features.shaderFloat16;
 
     std::string device_name = props2.properties.deviceName.data();
-    GGML_LOG_DEBUG("ggml_vulkan: %d = %s (%s) | uma: %d | fp16: %d | warp size: %d\n",
-              idx, device_name.c_str(), driver_props.driverName, uma, fp16, subgroup_size);
+    GGML_LOG_DEBUG("ggml_vulkan: %ld = %s (%s) | uma: %d | fp16: %d | warp size: %ld\n",
```

Comment:
For size_t I think it should be `%zu`.

Reply:
You're right, I wasn't aware of that specifier. Fixed.